internal/repository/article/article.go

```diff
@@ -0,0 +1,85 @@
+package article
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"news/internal/entity"
+	"news/internal/repository/mgdb"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type ArticleRep struct {
+	articleTitle *mongo.Collection
+	article      *mongo.Collection
+}
+
+func NewArticleRep(client *mgdb.MongoDriver) *ArticleRep {
+	return &ArticleRep{
+		articleTitle: client.CreateCollection("news", "articleTitle"),
+		article:      client.CreateCollection("news", "article"),
+	}
+}
+
+func (a *ArticleRep) SaveArticleList(ctx context.Context, articleList *entity.ArticleList) ([]int64, error) {
+	articles := make([]interface{}, 0, len(articleList.ArticleList.Articles))
+	for _, article := range articleList.ArticleList.Articles {
+		articles = append(articles, article)
+	}
+	insertedIds, err := a.articleTitle.InsertMany(ctx, articles)
+	if err != nil {
+		return nil, err
```

Comment:
it's a good practice to use `fmt.Errorf` with a custom prefix to know where error comes from.

Reply:
fixed